### PR TITLE
fix some comments

### PR DESCRIPTION
**Pull request by cuishuang**
## Checklist

- [x] I have linked to any relevant issues.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [x] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [x] I have requested a review from the relevant team or maintainers.

### CODE REVIEW

**Review by sdankel — APPROVED** [review]

**Review by IGI-111 — DISMISSED** [review]

File: forc-pkg/src/manifest/mod.rs [hunk 1]
```diff
@@ -437,7 +437,7 @@ impl PackageManifestFile {
         pkg_dir.pop();
         if let Some(nested_package) = find_nested_manifest_dir(&pkg_dir) {
             // remove file name from nested_package_manifest
-            bail!("Nested packages are not supported, please consider seperating the nested package at {} from the package at {}, or if it makes sense consider creating a workspace.", nested_package.display(), pkg_dir.display())
+            bail!("Nested packages are not supported, please consider separating the nested package at {} from the package at {}, or if it makes sense consider creating a workspace.", nested_package.display(), pkg_dir.display())
```
Comment (IGI-111) [inline forc-pkg/src/manifest/mod.rs:440]: This isn't a comment, it's an actual error message.
The test for this needs to be updated as well.

**Review by kayagokalp — CHANGES_REQUESTED** [review]
Hi, thanks for the PR. CI seems to be failing because you updated an error message but did not changed the test for that error message. So what you need to do make the pipeline green is changing the expected message at [the related test.toml](https://github.com/FuelLabs/sway/blob/master/test/src/e2e_vm_tests/test_programs/should_fail/forc/nested_package/test.toml).

**Review by kayagokalp — APPROVED** [review]
Thanks!

**Review by IGI-111 — APPROVED** [review]

**Comment by IGI-111** [comment]
Solve the merge conflicts and we should be able to merge this.

**Review by kayagokalp — APPROVED** [review]

**Review by IGI-111 — APPROVED** [review]

**Comment by IGI-111** [comment]
It's outdated again :sweat:

**Review by xunilrj — APPROVED** [review]

**Comment by IGI-111** [comment]
The HEAD is moving pretty fast so it's outdated again, please check the box that [allows edits by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) in this PR so that we can auto update it and it can be merged.

**Review by sdankel — APPROVED** [review]

**Comment by IGI-111** [comment]
I'm going to close this PR, please reopen one with a personal account fork and we'll get this merged.

**Pull request closed without merging**